packages/makeup-hoist-element/src/index.js

```diff
@@ -0,0 +1,79 @@
+let modalEl;
+let modalPlaceholder;
+let containerDiv;
+let prevAriaHiddenSetting;
+let bodyIndexes = [];
```

Comment:
Can we discuss on the naming of these? It's a bit hard to understand. 
* `modalEl` makes sense.
* `modalPlaceholder` seems okay, this is what remembers the old modal position? Maybe call it something like oldModalPositionEl?
* `containerDiv` again not sure what this is? 
* `prevAriaHiddenSettings` it's fine, but is this the value or a setting? If it's just the value true/false, then call it value not settings. It seems like it means it's a configuration
* `bodyIndexes`, again, not sure what this variable is? Maybe get a better name or comment what the point of this is?

Can we call the element that holds the modal in the body something like: `hoistedModalEl` or something (I'm open)? So we know it's the element holding the modal in the body?

Reply:
hoist-element module should have no knowledge of how it is being used, so let's keep the naming "modal"-free.